CONTRIBUTING.md

```diff
@@ -31,6 +31,42 @@ The Dart source code in this repo follows the:
 
 You should familiarize yourself with those guidelines.
 
+## Pubspec versioning
+
+The first change to a package after a publish needs to increment the package's
+pubspec version. Additional changes may also require pubspec version updates;
+see our
+[package maintenance wiki](https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#making-a-change)
+for a description of the process.
+
+## Changelog entries
+
+Most changes should be accompanied by a new entry in the CHANGELOG.md file. A
+good rule of thumb is that if the change is user-facing, it should have a
+changelog entry. This includes changes to documentation - like a README.md file
+update - or changes to dartdoc documentation.
+
+Some changes which are exempt are:
```

Comment:
Why explicitly name these exceptions? Do they not follow from the description above? Also, `some changes` leaves ambiguity, so I would rather just not talk about exceptions and leave that to common sense.